config/templates/package-scripts/postrm.erb

```diff
@@ -41,9 +41,32 @@ purge_sensu_files()
   fi
 }
 
+# only execute this script on rhel-based distros if an rpm uninstall
```

Comment:
I suggest that the platform-specific comments be moved inside the case switch so they are only printed on platforms where the code they apply to would be present.